cfme/web_ui/__init__.py

```diff
@@ -2039,6 +2039,15 @@ def name(self):
         """ Returns name of the quadicon."""
         return self._name
 
+    @property
```

Comment:
I'm not 100% sold on the name of this method. Perhaps if I am understanding it correctly, we could call it, ```icon_only``` or ```center_icon_only```.